Lib/fontParts/objects/base/validators.py

```diff
@@ -115,10 +115,11 @@ def validateLayerName(value):
 # -----
 
 def validateGlyphName(value):
-    """
-    XXX implement
-    """
-    return value
+    if not isinstance(value, basestring):
+        raise FontPartsError("Glyph names must be unicode strings, not %s." % type(value).__name__)
```

Comment:
I'd just change the error message to say "string" instead of "unicode strings", which is what the code already does: ie. accept either str (bytes) or unicode string values but coercing the former to unicode(value) using the default ascii encoding.